ckan/controllers/group.py

```diff
@@ -295,15 +305,17 @@ def pager_url(q=None, page=None):
                     facets[facet] = facet
 
             # Facet titles
-            for plugin in plugins.PluginImplementations(plugins.IFacets):
-                if self.group_type == 'organization':
+            from ckan.controllers.organization import OrganizationController
+            if isinstance(self, OrganizationController):
```

Comment:
The choice between group_facets and organization_facets is down to whether we're in GroupController or OrganizationController. Ideally this would be a method which OrganizationController overrides, but it is neater to keep it inline. So we simply use isinstance.